docs/content/fichier.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "1Fichier"
+description: "Rclone docs for 1Fichier"
+date: "2015-10-14"
+---
+
+<i class="fa fa-archive"></i> 1Fichier
+-----------------------------------------
+
+Paths are specified as `remote:path`
+
+Paths may be as deep as required, eg `remote:directory/subdirectory`.
+
+The initial setup for 1Fichier involves getting the API key from the website which you
+need to do in your browser.
+
+Here is an example of how to make a remote called `remote`.  First run:
+
+     rclone config
+
+This will guide you through an interactive setup process:
+
+```
+No remotes found - make a new one
+n) New remote
+s) Set configuration password
+q) Quit config
+n/s/q> n
+name> remote
+Type of storage to configure.
+Enter a string value. Press Enter for the default ("").
+Choose a number from below, or type in your own value
+  1 / 1Fichier
+    \ "fichier"
+...
+ 28 / http Connection
+    \ "http"
+Storage> fichier
+** See help for fichier backend at: https://rclone.org/fichier/ **
+
+Your API Key, get it from https://1fichier.com/console/params.pl
+Enter a string value. Press Enter for the default ("").
+api_key> example_key
+
+Edit advanced config? (y/n)
+y) Yes
+n) No
+y/n> 
+Remote config
+--------------------
+[remote]
+type = fichier
+api_key = example_key
+--------------------
+y) Yes this is OK
+e) Edit this remote
+d) Delete this remote
+y/e/d> y
+```
+
+
+Once configured you can then use `rclone` like this,
+
+List directories in top level of your Box
+
+    rclone lsd remote:
+
+List all the files in your Box
+
+    rclone ls remote:
+
+To copy a local directory to an Box directory called backup
+
+    rclone copy /home/source remote:backup
+
+### Modified time and hashes ###
+
+1Fichier does not support modification times. It supports the Whirlpool hash algorithm.
+
+### Duplicated files ###
+
+1Fichier can have two files with exactly the same name and path (unlike a
+normal file system).
+
+Duplicated files cause problems with the syncing and you will see
+messages in the log about duplicates.
+
+Use `rclone dedupe` to fix duplicated files.
```

Comment:
Does `rclone dedupe` work? It needs to support `MergeDirs` to do that

That could be another issue for later.